core/src/peer_id.rs

```diff
@@ -24,10 +24,6 @@ use quick_error::quick_error;
 use multihash;
 use std::{convert::TryFrom, fmt, str::FromStr};
 
-/// Public keys with byte-lengths smaller than `MAX_INLINE_KEY_LENGTH` will be
-/// automatically used as the peer id using an identity multihash.
-const MAX_INLINE_KEY_LENGTH: usize = 42;
```

Comment:
I think that should not be removed, but may be commented out, referring to the related comment that can be found in `PeerId::from_public_key` (where the code that uses it is commented out). This code path will eventually be enabled in an upcoming release. See https://github.com/libp2p/rust-libp2p/pull/1237/files.

Reply:
Awesome, thanks for the heads up. Reverted. 👍